api/src/test/java/org/openmrs/validator/ConceptValidatorTest.java

```diff
@@ -461,9 +471,81 @@ public void validate_shouldPassIfDifferentConceptsHaveTheSameShortNames() throws
 		
 		concept.addName(conceptFullySpecifiedName);
 		concept.addName(conceptShortName);
+		concept.addDescription(new ConceptDescription("some description",Context.getLocale()));
 		
 		Errors errors = new BindException(concept, "concept");
 		new ConceptValidator().validate(concept, errors);
 		Assert.assertEquals(false, errors.hasErrors());
 	}
+
+	/**
+	 * @see ConceptValidator#validate(Object,Errors)
+	 **/
+	@Test
+	@Verifies(value = "fail if any description is a not entered while creating a new concept ", method = "validate(Object,Errors)")
+	public void validate_shouldFailIfAnyDescriptionIsNotEnteredWhileCreatingANewConcept() throws Exception {
+		Concept concept = new Concept();
+		concept.addName(new ConceptName("some name", Context.getLocale()));
+		Errors errors = new BindException(concept, "concept");
+		new ConceptValidator().validate(concept, errors);
+		Assert.assertTrue(errors.hasFieldErrors("descriptions"));
+	}
+
+	/**
+	 * @see ConceptValidator#validate(Object,Errors)
+	 **/
+	@Test
+	@Verifies(value = "fail if any description is a removed while editing a new concept ", method = "validate(Object,Errors)")
```

Comment:
@wluyima 
Removed this test case as I am not generating any errors if a user tries to save concept with empty description.
I am checking for atleast one description in any locale and removing the empty descriptions in ConceptSaveHandler.